packages/big-bang-core/README.md

```diff
@@ -15,60 +15,100 @@ This package deploys [Big Bang Core](https://repo1.dso.mil/platform-one/big-bang
 
 ## Instructions
 
-### Pull down the code and binaries
+### Pull down the code and binaries [local machine]
 
 ```shell
-# clone the binaries
+# Clone the binaries
 git clone https://github.com/defenseunicorns/zarf.git
 
-# change to the examples folder
-cd zarf/examples
+# Change dir
+cd zarf
+```
+
+### Get Zarf components [local machine]
+
+Follow instructions on [this page](../../docs/5-operator-manual/0-set-up-and-install.md) to get the `zarf` cli and the `zarf-init*.tar.zst` package and place them in the ./build directory
 
-# Download the latest release of Zarf and the Init Package to the 'examples/sync' folder
-make fetch-release
+Alternatively, you could build the components from the repo
+```shell
+# Build zarf components from scratch (NOTE: golang and npm must be installed)
+make init-package
 ```
 
-### Build the deploy package
+### Build the deploy package [local machine]
 
 ```shell
-# Create the deploy package and move it to the 'examples/sync' folder
-make package-example-big-bang
+# Change dir
+cd packages/big-bang-core
+
+# Authenticate to the registry with Big Bang artifacts
+set +o history
+export REGISTRY1_USERNAME=<REPLACE_ME>
+export REGISTRY1_PASSWORD=<REPLACE_ME>
+echo $REGISTRY1_PASSWORD | ../../build/zarf tools registry login registry1.dso.mil --username $REGISTRY1_USERNAME --password-stdin
+set -o history
+
+# Run zarf package command
+../../build/zarf package create . --confirm
 ```
 
-### Start the Vagrant VM
+### Start the Vagrant VM [local machine]
```

Comment:
I have a preference to completely delete this whole VM section as it's super gross and out of date. I'd prefer to just do K3d instead. Would be much easier to test against and don't see any real drawback.

Reply:
@jeff-mccoy completely agree, had to do some pretty uncouth stuff to make it work with the vagrant VM approach.  will take another crack using k3d instead.

Reply:
@jeff-mccoy now the docs have been updated with usage of k3d instead of the vagrant vm.